tests/unit/EnumMethodReflectionTest.php

```diff
@@ -155,21 +165,11 @@ public function testHasSideEffects(): void
 
     public static function assertMatchesRegularExpression(string $pattern, string $string, string $message = ''): void
     {
-        if (method_exists(parent::class, 'assertMatchesRegularExpression')) {
-            parent::assertMatchesRegularExpression($pattern,  $string, $message);
-            return;
-        }
-
-        self::assertRegExp($pattern, $string, $message);
+        parent::assertMatchesRegularExpression($pattern, $string, $message);
     }
 
     public static function assertDoesNotMatchRegularExpression(string $pattern, string $string, string $message = ''): void
     {
-        if (method_exists(parent::class, 'assertDoesNotMatchRegularExpression')) {
-            parent::assertDoesNotMatchRegularExpression($pattern,  $string, $message);
-            return;
-        }
-
-        self::assertNotRegExp($pattern, $string, $message);
+        parent::assertDoesNotMatchRegularExpression($pattern, $string, $message);
     }
```

Comment:
`assertNotRegExp` got removed in PHPUnit 9 and `assertDoesNotMatchRegularExpression` should be used instead.
PHPUnit 9 supports PHP >= 7.3.

-> Please drop PHPUnit < 9 and this method can safely be removed as it will be available on the parent.